doc/userguide/src/SupportingTools/Libdoc.rst

```diff
@@ -41,22 +41,22 @@ Synopsis
 Options
 ~~~~~~~
 
-  -f, --format <html|xml> Specifies whether to generate an HTML or XML
-                          output file. The default output `format` and the 
-                          default output `specdocformat` is got from the
-                          output file extension.
-                          :file:`*.html`    -> `-f HTML -s HTML`
-                          :file:`*.xml`     -> `-f XML  -s RAW`
-                          :file:`*.libspec` -> `-f XML  -s HTML`
-   -s --specdocformat <raw|html>
+  -f, --format <html|xml>
+                          Specifies whether to generate an HTML, XML or
+                          JSON output file. The default output `format`
+                          and the default output `specdocformat` is fetched
+                          from the output file extension.
+                          :file:`*.html` -> `-f HTML -s HTML`,
+                          :file:`*.xml` -> `-f XML -s RAW`,
+                          :file:`*.json` -> `-f JSON -s HTML` and
+                          :file:`*.libspec` -> `-f XML -s HTML`
```

Comment:
Explanation related to default spec doc format doesn't really belong here. It's not a directly related option and isn't even applicable with the HTML format. The end of the documentation could be something like this:

---

The default `format` is got based on the output file extension as follows:

- `*.html` -> HTML
- `*.xml` -> XML
- `*.json` -> JSON
- `*.libspec` -> XML (with HTML spec documentation format)

---

Reply:
i think it is handy to have it here, because file extension and `--format` is directly affecting the `--specdocformat`. So this is the most direct place to say that. As in command line as well.
Users can imho easily understand how these two options affect each other.

But i am totally fine if you want to restructure the docs otherwise.
I think this is more a question of taste so maybe you can put it where you think it is better.